README.md

```diff
@@ -1057,6 +1092,22 @@ for the client-cluster connection, you should set an SSL configuration. Please s
 
 As explained in the [TLS/SSL section](#61-tlsssl), Hazelcast members have key stores used to identify themselves (to other members) and Hazelcast C++ clients have certificate authorities used to define which members they can trust. 
 
+## 5.8. Enabling Hazelcast AWS Cloud Discovery
+
+The C++ client can discover the exiting Hazelcast servers in the Amazon AWS environment. The client queries the Amazon AWS environment using the [describe-instances] (http://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html) query of AWS. The client finds only the up and running instances and filters them based on the filter config provided at the ClientAwsConfig configuration.
```

Comment:
If an instance is running, isnt it already up?  `The client finds only the up and running instances` -> `The client finds only the running instances`

Reply:
The AWS instance may be up but there may be no Hazelcast server process running.